backend/src/main/resources/db/migration/V1.23__add_some_se_and_sepd_indexes.sql

```diff
@@ -0,0 +1,33 @@
+-- Add index on (accession, version) without pipeline_version for faster lookups
+-- This improves query performance on sequence_entries_view which joins on these columns
+DROP INDEX IF EXISTS sequence_entries_preprocessed_data_accession_version_idx;
```

Comment:
this doesnt seem to currently exist? we only have sequence_entries_preprocessed_data_pkey (which we should keep)